dbt_af/operators/base.py

```diff
@@ -65,13 +66,18 @@ def __init__(
         kwargs.update(get_delay_by_schedule(schedule_tag))
         af_pool = pool or f'dbt_{self.target_environment}' if dbt_af_config.use_dbt_target_specific_pools else None
 
+        retry_policy = (
+            self.retry_policy.as_dict()
+            if self.retry_policy is not None
+            else dbt_af_config.retries_config.default_retry_policy
+        )
```

Comment:
Maybe wrap it in property?

Reply:
Removed attributes and moved them to arguments, because the exact retry policy is resolved in each operation class.